x-pack/plugins/observability/public/pages/alert_details/components/header_actions.tsx

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { noop } from 'lodash';
+import { CaseAttachmentsWithoutOwner } from '@kbn/cases-plugin/public/types';
+import { CommentType } from '@kbn/cases-plugin/common';
+import { EuiButton, EuiButtonEmpty, EuiFlexGroup, EuiPopover, EuiText } from '@elastic/eui';
+import { ALERT_RULE_UUID, ALERT_UUID } from '@kbn/rule-data-utils';
+
+import { useKibana } from '../../../utils/kibana_react';
+import { useFetchRule } from '../../../hooks/use_fetch_rule';
+import { ObservabilityAppServices } from '../../../application/types';
+import { TopAlert } from '../../alerts';
+
+export interface HeaderActionsProps {
+  alert: TopAlert | null;
+  externalConnector?: {
+    name: string;
+    onViewInExternalApp: (alertId: string) => void;
```

Comment:
If I am not wrong, this functionality is same as `View in app` in Alerts table. The idea now is to display app name instead of just `app`. For example, for APM rule alerts, in place of `View in app`, it should say `View in APM`.

<img width="322" alt="Screenshot 2022-09-30 at 11 05 09" src="https://user-images.githubusercontent.com/69037875/193235471-35b6b1a0-b3d8-43c1-8c8f-ce399c75ad3c.png">

Reply:
@maciejforcone has told me it should be as it is now implemented in the code.

Reply:
Maybe I am missing something. Which app `onViewInExternalApp` will open?

Reply:
Confirmed with @maciejforcone

Reply:
probably my comment above is missed. Could you let me know which app/connector it will redirect user to? Is it something we are going to implement in next phase therefore out of scope for now?

Reply:
@benakansara @fkanout I am not sure how to answer that. @maciejforcone has mentioned that he has an idea how this should be implemented.

Reply:
@CoenWarmer, I think this is something that would be added later, it rings a bell, and for sure it's not the same as View In-app

Reply:
@fkanout I see, thanks for the clarification.